app/main/plugins/core/basic-apps/index.js

```diff
@@ -16,6 +16,7 @@ const fn = ({ term, actions, display }) => {
     return {
       icon,
       id: id || path,
+      count: app.selectCount || 0,
```

Comment:
`count` key will be just ignored. I'd not add new key for public plugin API to only sort results.